packages/protocol-contracts/contracts/ZetaInteractor.sol

```diff
@@ -46,7 +47,7 @@ abstract contract ZetaInteractor is Ownable, ZetaInteractorErrors {
      * @dev Useful for contracts that inherit from this one
      */
     function _isValidChainId(uint256 chainId) internal view returns (bool) {
-        return (keccak256(interactorsByChainId[chainId]) != keccak256(new bytes(0)));
+        return (keccak256(interactorsByChainId[chainId]) != ZERO_BYTES);
```

Comment:
Do we need `keccak256` here?

Reply:
yes, because we need to cast to memory byte32